deploy/03-rbac.yaml

```diff
@@ -0,0 +1,72 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-samples-operator
+  namespace: openshift-cluster-samples-operator
+rules:
+- apiGroups:
+  - samplesoperator.config.openshift.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
+
+kind: ClusterRole
```

Comment:
missing a --- above this.

Reply:
ah ... thought those were optional

Reply:
you're combining multiple resources into a single file, so you need to split them.